lighthouse-core/scripts/release/test.sh

```diff
@@ -23,9 +23,6 @@ yarn build-all
 echo "Running the standard test suite..."
 yarn test
 
-echo "Running the smoke tests...."
-yarn smoke || yarn smoke || yarn smoke
```

Comment:
this is not really necessary.
- in theory, master is green. `yarn test` is a nice sanity check, but running all the smoke tests is overkill because
- they're about to be run again in the packaged pristine version and
- it's incredibly annoying to have to run this all over again if master is still green but there's some hiccup further down in this script :)